### ISSUE TITLE
Is this a BTE Predicted Edge?

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#396]
This edge for result Prednisolone, from the BTE Support Graph has two sources:

https://ui.ci.transltr.io/results?l=Bethlem%20Myopathy&i=MONDO:0008029&t=0&q=1fb6945c-668b-4750-85f4-2daa53eb4596
https://arax.ci.transltr.io/?r=1fb6945c-668b-4750-85f4-2daa53eb4596

BTE
Monarch Initiative

I thought BTE was a source when the edge is predicted.  I'm not sure this edge is predicted, its fairly easy to look up in Monarch, and in the ARAX UI I can't see the EPC on the second edge.  

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/dcc7f4c0-f93d-43f7-b0d1-e31e004b5873)

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/45e91401-fee6-446b-bea6-5392498729de)

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/81da4d51-aded-4b8e-8ab1-8317ff63acd9)

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/6b19b0a2-af8d-4b91-91c8-550dfa2764a4)

**Comment by andrewsu** [comment]
> I thought BTE was a source when the edge is predicted. I'm not sure this edge is predicted, its fairly easy to look up in Monarch, and in the ARAX UI I can't see the EPC on the second edge.

**TLDR:** BTE is the primary knowledge source on the _inferred_ edge `COL6A1 - causes - Bethlem myopathy`, which is supported by two edges `COL6A1 - causes - Bethlem myopathy 1` (PKS: MonarchInitiative) and `Bethlem myopathy 1 - subclass of - Bethlem myopathy` (PKS: MONDO).  Note that in the screenshot above, Monarch Initiative does _not_ make the direct assertion, which is why BTE has to infer it through subclassing and hence claim to be the primary knowledge source.

@sstemann If this explanation/behavior makes sense to you, then feel free to close the issue.  If you think the UI should display this info differently, then feel free to add more to this issue.

More details below...

---------------


Unpacking this issue starting from the BTE response https://arax.ci.transltr.io/api/arax/v1.4/response/3d50f996-3e7e-43fa-92b6-01aacd1e88b1

### Nodes
* Prednisolone is `PUBCHEM.COMPOUND:5755`
* COL6A1 is `NCBIGene:1291`
* Bethlem myopathy is `MONDO:0008029`
* Bethlem myopathy 1 is `MONDO:0024530`
* Bethlem myopathy 2 is `MONDO:0034022`

### Result and edges

BTE returns one result object saying `prednisolone treats Bethlem myopathy`:

```
      {
        "analyses": [
          {
            "edge_bindings": {
              "t_edge": [
                {
                  "id": "inferred-PUBCHEM.COMPOUND:5755-treats-MONDO:0008029"
                }
              ]
            },
            "resource_id": "infores:biothings-explorer",
            "score": 0.9461577440460234
          }
        ],
        "node_bindings": {
          "on": [
            {
              "id": "MONDO:0008029"
            }
          ],
          "sn": [
            {
              "id": "PUBCHEM.COMPOUND:5755"
            }
          ]
        },
        "normalized_score": 94.4,
        "resource_id": "BTE"
      },
```

This is the edge binding, which has two support graphs
```
        "inferred-PUBCHEM.COMPOUND:5755-treats-MONDO:0008029": {
          "attributes": [
            {
              "attribute_type_id": "biolink:support_graphs",
              "value": [
                "inferred-PUBCHEM.COMPOUND:5755-treats-MONDO:0008029-support0",
                "inferred-PUBCHEM.COMPOUND:5755-treats-MONDO:0008029-support1"
              ]
            }
          ],
          "object": "MONDO:0008029",
          "predicate": "biolink:treats",
          "sources": [
            {
              "resource_id": "infores:biothings-explorer",
              "resource_role": "primary_knowledge_source"
            }
          ],
          "subject": "PUBCHEM.COMPOUND:5755"
        },
```
This is the first of the support graphs `inferred-PUBCHEM.COMPOUND:5755-treats-MONDO:0008029-support0`:
```
      "inferred-PUBCHEM.COMPOUND:5755-treats-MONDO:0008029-support0": {
        "edges": [
          "2f8e939b720d1c596f0c8dde26f97202",
          "NCBIGene:1291-causes-MONDO:0008029-via_subclass",
          "f95f24ec12391532a1b7f1238d628c98"
        ]
      },
```

This support graph has three cited edges:

* `2f8e939b720d1c596f0c8dde26f97202`: an edge for `COL6A1 - causes - Bethlem myopathy` with primary knowledge source of Monarch Initiative
* `NCBIGene:1291-causes-MONDO:0008029-via_subclass`: another _inferred_ edge for `COL6A1 - causes - Bethlem myopathy` with primary knowledge source of BTE. The supporting graph underlying this inferred edge has two edges:
     * `e53d1ab5e124d4072f5ecea65f099b5c`: `COL6A1 - causes - Bethlem myopathy 1` (PKS: MonarchInitiative)
     * `expanded-MONDO:0024530-subclass_of-MONDO:0008029`: `Bethlem myopathy 1 - subclass of - Bethlem myopathy` (PKS: MONDO)
* `f95f24ec12391532a1b7f1238d628c98`: An edge for `prednisolone - affects - COL6A1` from TMKP

<details>
<summary>TRAPI snippets</summary>

```
        "2f8e939b720d1c596f0c8dde26f97202": {
          "attributes": [
            {
              "attribute_type_id": "biolink:original_predicate",
              "value": [
                "contributes to condition"
              ]
            },
            {
              "attribute_type_id": "monarch_source_database",
              "value": [
                "https://archive.monarchinitiative.org/#omim",
                "https://archive.monarchinitiative.org/#orphanet"
              ]
            }
          ],
          "object": "MONDO:0008029",
          "predicate": "biolink:causes",
          "sources": [
            {
              "resource_id": "infores:monarchinitiative",
              "resource_role": "primary_knowledge_source"
            },
            {
              "resource_id": "infores:biolink-api",
              "resource_role": "aggregator_knowledge_source",
              "upstream_resource_ids": [
                "infores:monarchinitiative"
              ]
            },
            {
              "resource_id": "infores:biothings-explorer",
              "resource_role": "aggregator_knowledge_source",
              "upstream_resource_ids": [
                "infores:biolink-api"
              ]
            }
          ],
          "subject": "NCBIGene:1291"
        },
```
```
        "NCBIGene:1291-causes-MONDO:0008029-via_subclass": {
          "attributes": [
            {
              "attribute_type_id": "biolink:support_graphs",
              "value": [
                "support0-NCBIGene:1291-causes-MONDO:0008029-via_subclass"
              ]
            }
          ],
          "object": "MONDO:0008029",
          "predicate": "biolink:causes",
          "sources": [
            {
              "resource_id": "infores:biothings-explorer",
              "resource_role": "primary_knowledge_source"
            }
          ],
          "subject": "NCBIGene:1291"
        },
```
```
        "f95f24ec12391532a1b7f1238d628c98": {
          "attributes": [
            {
              "attribute_source": "infores:text-mining-provider-targeted",
              "attribute_type_id": "biolink:primary_knowledge_source",
              "value": "infores:text-mining-provider-targeted",
              "value_type_id": "biolink:InformationResource"
            },
            [... snip...]
          ],
          "object": "NCBIGene:1291",
          "predicate": "biolink:affects",
          "qualifiers": [
            {
              "qualifier_type_id": "biolink:qualified_predicate",
              "qualifier_value": "biolink:causes"
            },
            {
              "qualifier_type_id": "biolink:object_aspect_qualifier",
              "qualifier_value": "activity_or_abundance"
            },
            {
              "qualifier_type_id": "biolink:object_direction_qualifier",
              "qualifier_value": "increased"
            }
          ],
          "sources": [
            {
              "resource_id": "infores:text-mining-provider-targeted",
              "resource_role": "primary_knowledge_source"
            },
            {
              "resource_id": "infores:biothings-explorer",
              "resource_role": "aggregator_knowledge_source",
              "upstream_resource_ids": [
                "infores:text-mining-provider-targeted"
              ]
            }
          ],
          "subject": "PUBCHEM.COMPOUND:5755"
```
</details>

**Comment by sierra-moxon** [comment]
@andrewsu - thank you for the explanation for BTE as a primary source.  I am not quite sure I understand why Monarch is showing up as a primary source though. 

in particular this in the TRAPI snippet where the "original_predicate" is "contributes to condition" and the "predicate" is "causes":
```
        "2f8e939b720d1c596f0c8dde26f97202": {
          "attributes": [
            {
              "attribute_type_id": "biolink:original_predicate",
              "value": [
                "contributes to condition"
              ]
            },
            {
              "attribute_type_id": "monarch_source_database",
              "value": [
                "https://archive.monarchinitiative.org/#omim",
                "https://archive.monarchinitiative.org/#orphanet"
              ]
            }
          ],
          "object": "MONDO:0008029",
          "predicate": "biolink:causes",
          "sources": [
            {
              "resource_id": "infores:monarchinitiative",
              "resource_role": "primary_knowledge_source"
            },
            {
              "resource_id": "infores:biolink-api",
              "resource_role": "aggregator_knowledge_source",
              "upstream_resource_ids": [
                "infores:monarchinitiative"
              ]
            },
            {
              "resource_id": "infores:biothings-explorer",
              "resource_role": "aggregator_knowledge_source",
              "upstream_resource_ids": [
                "infores:biolink-api"
              ]
            }
          ],
          "subject": "NCBIGene:1291"
        },
```

**Comment by sierra-moxon** [comment]
I also went spelunking in the biolink-api and I don't see "causes" as a predicate for Bethlem myopathy (instead for Bethlem myopathy **1** - Monarch should not be the PKS for the 'causes' edge to Bethlem myopathy?):

[https://api.monarchinitiative.org/api/bioentity/gene/NCBIGene%3A1291/diseases?rows=100&fa[…]se&direct=false&direct_taxon=false&association_type=both](https://api.monarchinitiative.org/api/bioentity/gene/NCBIGene%3A1291/diseases?rows=100&facet=false&unselect_evidence=false&exclude_automatic_assertions=false&fetch_objects=false&use_compact_associations=false&direct=false&direct_taxon=false&association_type=both)

**Comment by andrewsu** [comment]
Based on the [SmartAPI annotation for the Monarch Initiative API](https://github.com/NCATS-Tangerine/translator-api-registry/blob/master/biolink/openapi.yml), this is the API call that BTE is making:

https://api.monarchinitiative.org/api/bioentity/gene/NCBIGene%3A1291/diseases?rows=-1&direct=true&unselect_evidece=true&association_type:causal

When `association_type:causal`, we map that to the predicate `caused_by` (based on the [disease-gene-1 operation](https://github.com/NCATS-Tangerine/translator-api-registry/blob/master/biolink/openapi.yml#L952)), whereas when `association_type:non-causal` gets mapped to the predicate `condition_associated_with_gene` (based on the [disease-gene-2 operation](https://github.com/NCATS-Tangerine/translator-api-registry/blob/master/biolink/openapi.yml#L988)). 

In this case, it is the second entry (see screenshot below) that BTE is using to claim that Monarch Initiative is a primary source for `COL6A1 - causes - Bethlem myopathy`. Presumably, the API is equating the original predicate "contributes to condition" (`RO:0003304`) with that `association_type:causal` API parameter, which we then equate with `biolink:causes`. Happy to adjust the SmartAPI annotation if we should be calling the API and/or interpreting API results differently...

![image](https://github.com/NCATSTranslator/Feedback/assets/2635409/c85bd6c6-44ce-4a70-a770-d2c4cb60842b)

**Comment by sstemann** [comment]
makes sense to close
